cddl/profile.cddl

```diff
@@ -1,3 +1,3 @@
 profile-claim = (
-    profile => tstr / oid
+    profile => ~uri / ~oid
```

Comment:
a curiosity: what is the rationale for unwrapping?

Reply:
You can tell uri from old by the CBOR type (text vs byte string) so no need for the tag. 

Generally, it seems the convention in CBOR is to unwrap if possible to not have redundancy (which creates an error condition) and to save bytes. If others think otherwise, fill me in. :-)

Reply:
makes sense, thanks for the explanation